output.go

```diff
@@ -146,6 +146,11 @@ func (o *Output) flushMetrics() {
 	sampleContainers := o.GetBufferedSamples()
 	start := time.Now()
 
+	o.logger.Debug("flushMetrics: Collecting...")
+	o.logger.WithField("sample-containers", len(sampleContainers)).Debug("flushMetrics: Collecting...")
+
+	rows := [][]interface{}{}
+
 	var batch pgx.Batch
```

Comment:
Nit: With the metrics samples no longer using the `Batch,` I'd prefer to see the original variable declaration moved just prior to the loop responsible for populating the batch (~line 166).